pnpm-workspace.yaml

```diff
@@ -1,10 +1,12 @@
 packages:
-  - 'benchmark'
-  - 'packages/*'
-  - 'packages/mui-envinfo/test'
-  - 'packages-internal/*'
-  - 'docs'
-  - 'test'
-  - 'test/moduleResolution'
-  - 'apps/*'
-  - 'scripts/sizeSnapshot'
+  - benchmark
+  - packages/*
+  - packages/mui-envinfo/test
+  - packages-internal/*
+  - docs
+  - test
+  - test/moduleResolution
+  - apps/*
+  - scripts/sizeSnapshot
+patchedDependencies:
+  styled-components: patches/styled-components.patch
```

Comment:
Was this patch meant to be added? 🤔

Reply:
yes, it's meant to replace `skipLibCheck`. It would be equally broken on end-users side, but at least we're not sweeping other errors under the rug.